### PR TITLE
updated mention about secret-to-public gist feature on GitHub

### LINKED ISSUE #25874: clarity on whether secret-to-public gist is possible
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists

### What part(s) of the article would you like to see updated?

The section `After creating a gist, you cannot convert it from public to secret.`, is not clear whether the reverse is true; that is, a secret gist can/cannot be converted to public. 

After checking numerous forums (with dated information), this `docs` repo's `Issues` and `Discussions`, it is not clear if an initially-created secret gist can be converted to public later on.

Suggesting to include some mention if a secret gist can/cannot be converted to public.

### Additional information

First-time contributing here ;)

### PR DESCRIPTION
**Pull request by PaoloJr**
### Why:
Details on updating a secret gist to be public are not clear in the docs.

Closes:  #25874

### What's being changed (if available, include any code snippets, screenshots, or gifs):
Added a mention that a secret gist may be made public by editing it (clicking "Make public"). The additional phrase is insterted right after `After creating a gist, you cannot convert it from public to secret.` sentence on the `https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists` page

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@PaoloJr90 Thanks for submitting a PR! I've edited your post slightly to connect it with your issue. 👍 

I'll get this triaged for review! :zap:

**Review by sophietheking — DISMISSED** [review]
@PaoloJr90 👋 Thanks for opening this PR and contributing to our Docs! ✨ I left one small suggestion. Please feel free to `@mention` me here to get this merged. :shipit:

File: content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Gists can be {% ifversion ghae %}internal{% else %}public{% endif %} or secret.
 
 Secret gists don't show up in {% data variables.gists.discover_url %} and are not searchable unless you are logged in and are the author of the secret gist. Secret gists aren't private. If you send the URL of a secret gist to {% ifversion ghae %}another enterprise member{% else %}a friend{% endif %}, they'll be able to see it. However, if {% ifversion ghae %}any other enterprise member{% else %}someone you don't know{% endif %} discovers the URL, they'll also be able to see your gist. If you need to keep your code away from prying eyes, you may want to [create a private repository](/repositories/creating-and-managing-repositories/creating-a-new-repository) instead.
 
-{% data reusables.gist.cannot-convert-public-gists-to-secret %}
+{% data reusables.gist.cannot-convert-public-gists-to-secret %}. However, a secret gist can be made public by editing the gist and updating the visbility to "Make public".
```
[the remark this answers is not in the document]
Comment (PaoloJr90) [reply content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md:26]: Looks good @sophietheking 👍

**Review by PaoloJr — COMMENTED** [review]
This looks good 👍 ⚡

**Review by sophietheking — APPROVED** [review]

**Pull request merged**